### PR TITLE
[FIX] add support for pylint 2.3.0 (https://github.com/PyCQA/pylint/c…

### PR DESCRIPTION
**Pull request by shepilov-vladislav**
…ommit/da67a9da682e51844fbc674229ff6619eb9c816a)

### CODE REVIEW

File: pylint_plugin_utils/__init__.py [hunk 1]
```diff
@@ -119,13 +119,18 @@ def suppress_message(linter, checker_method, message_id_or_symbol, test_func):
...
+        get_message_definitions = msgs_store.get_message_definition
     else:
-        get_message_definition = msgs_store.get_message_definition
+        get_message_definitions = msgs_store.get_message_definitions
 
     try:
-        pylint_message = get_message_definition(message_id_or_symbol)
+        pylint_message = get_message_definitions(message_id_or_symbol)
+        pylint_message = pylint_message[0] if isinstance(pylint_message, (list, tuple)) else pylint_message
         symbols = [s for s in (pylint_message.msgid, pylint_message.symbol) if s is not None]
```
Comment (craiga) [inline pylint_plugin_utils/__init__.py]: Should `symbols` be built from each message definition returned by `get_message_definitions`?

```python
if isinstance(pylint_message, (list, tuple)):
    symbols = []
    for message in pylint_message:
        symbols.append(message.msgid)
        symbols.append(message.symbol)
```

**Comment by shepilov-vladislav** [comment]
@carlio Maybe new release from latest master?

**Comment by carlio** [comment]
@shepilov-vladislav already done - 0.5 is now on PyPI https://pypi.org/project/pylint-plugin-utils/ - just need to fix up pylint-django too

**Comment by carlio** [comment]
(though that has pylint-plugin-utils>0.4 so should be fine with new builds)

**Pull request merged**